src/bun.js/bindings/BunProcess.cpp

```diff
@@ -30,8 +30,11 @@
 #include <sys/utsname.h>
 #else
 #include <uv.h>
+// https://github.com/nodejs/node/blob/62ca05017d758e254eda2dfaf9678d2038a9235b/src/node_process_methods.cc#L28
```

Comment:
The comment url for Umask is not next to the line for defining umask. Could this be moved closer with added comment for context of the link.

Reply:
updated